modulegen/internal/vscode/types.go

```diff
@@ -1,34 +1,43 @@
 package vscode
 
-import "path/filepath"
+import (
+	"sort"
+)
 
-type ProjectDirectories struct {
-	RootDir            string
-	ModuleGeneratorDir string
-	Examples           []string
-	Modules            []string
+type Config struct {
+	Folders []Folder `json:"folders"`
 }
 
-func newProjectDirectories(rootDir string, examples []string, modules []string) *ProjectDirectories {
-	rootDirAbs, err := filepath.Abs(rootDir)
-	if err != nil {
-		rootDirAbs = rootDir
-	}
-
-	moduleGeneratorDirAbs := filepath.Join(rootDirAbs, "modulegen")
+type Folder struct {
+	Name string `json:"name"`
+	Path string `json:"path"`
+}
 
-	for i, example := range examples {
-		examples[i] = filepath.Join(rootDirAbs, "examples", example)
+func newConfig(examples []string, modules []string) *Config {
+	config := Config{
+		Folders: []Folder{
+			{
+				Path: "../",
+				Name: "core",
```

Comment:
```suggestion
				Name: "testcontainers-go",
```

Reply:
We probably need to hack the sort process

Reply:
OK, It's done ;)